src/components/Uploader.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import firebase from 'firebase';
+import FileUploader from 'react-firebase-file-uploader';
+
+class Uploader extends Component {
+  state = {
+    isUploading: false,
+  };
+
+  handleUploadStart = () => {
+    this.setState({ isUploading: true });
+  };
+
+  handleChangeImage = e => {
```

Comment:
should we avoid single letter variables?

Reply:
It's probably fine here. Using `e` for an event is pretty standard in JS. If you're strongly opposed, I can change to `event`

Reply:
yeah most of the examples/tutorials use 'e'. Just wanted to confirm :)